Randolph-dev/Druid_Partnering_Project_React24k
# Scrum 142 style the jobspage

## Related ticket:
[SCRUM-142](https://druidpartneringproject2024bch.atlassian.net/browse/SCRUM-142?atlOrigin=eyJpIjoiYWJiZWUyMjUyZTE2NDQ5NWJiYWNmODg3OGM1NzQ5YWYiLCJwIjoiaiJ9)

## In this PR:
- Display paragraph content on Jobspage
- Create sections Hero, Benefits, and About Company and adjust them according to the layout
- Some styling

## Testing instruction
- Checkout branch
- Import db and config
- Check the Jobspage


[SCRUM-142]: https://druidpartneringproject2024bch.atlassian.net/browse/SCRUM-142?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
